iml-agent/src/action_plugins/snapshot.rs

```diff
@@ -0,0 +1,199 @@
+            },
+            LdevEntry {
+                primary: "oss1".into(),
+                failover: Some("oss2".into()),
+                label: "zfsmo-OST0007".into(),
+                device: "zfs:ost7/ost7".into(),
+            },
+            LdevEntry {
+                primary: "oss1".into(),
+                failover: Some("oss2".into()),
+                label: "zfsmo-OST0008".into(),
+                device: "zfs:ost8/ost8".into(),
+            },
+            LdevEntry {
+                primary: "oss1".into(),
+                failover: Some("oss2".into()),
+                label: "zfsmo-OST0009".into(),
+                device: "zfs:ost9/ost9".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST000a".into(),
+                device: "zfs:ost10/ost10".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST000b".into(),
+                device: "zfs:ost11/ost11".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST000c".into(),
+                device: "zfs:ost12/ost12".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST000d".into(),
+                device: "zfs:ost13/ost13".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST000e".into(),
+                device: "zfs:ost14/ost14".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST000f".into(),
+                device: "zfs:ost15/ost15".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST00010".into(),
+                device: "zfs:ost16/ost16".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST00011".into(),
+                device: "zfs:ost17/ost17".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST00012".into(),
+                device: "zfs:ost18/ost18".into(),
+            },
+            LdevEntry {
+                primary: "oss2".into(),
+                failover: Some("oss1".into()),
+                label: "zfsmo-OST00013".into(),
+                device: "zfs:ost19/ost19".into(),
```

Comment:
Do you think more than two entries are necessary?

Reply:
My personal preference was to generate the config file in the ticket. It doesn't hurt either way though.